src/bttrx_fsm.cpp

```diff
@@ -22,6 +22,8 @@ Contact: bt-trx.com, mail@bt-trx.com
 #include "resulttype.h"
 #include "splitstring.h"
 
+extern Preferences preferences;
+
```

Comment:
Please do not use Preferences directly here.
Please implement a getter method for the callsign value in the BTTRX_Control class.

Reply:
[x] done. Will be included in the PR against the dev branch.